### ISSUE TITLE
/status/1xx results in failures

### ISSUE THREAD

**Issue by cbeamond** [mccutchen/go-httpbin#48]
/status/1xx results in failures when tested. They are unusual codes but should be testable regardless since they are in the spec.

**Comment by mccutchen** [comment]
Yep!  This is gonna take a bit more digging on my end, as expected client and server behavior in the face of a 100 Continue response, and how we might realistically support 101 Switching Protocols.
